Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md

```diff
@@ -1,7 +1,7 @@
 # Create resources for the AKS Priavte Cluster
 
 The following will be created:
-* AKS Private Cluster
+* AKS Private Clusters
 * Log Analytics Workspace
```

Comment:
I also thought that we were going to have separate .md files for Blue/Green deployment? I thought the idea was to use the same code base but this would be a separate scenario to follow. We have to be careful here not to confuse people by going too advanced straight away. This should be targeting people that are relatively new to AKS and want to get started. I think if we throw in Blue/green straight away it's going to overload them. Let's discuss this separately also.